icu4j/main/classes/core/src/com/ibm/icu/util/JapaneseCalendar.java

```diff
@@ -634,6 +414,7 @@ protected int handleGetLimit(int field, int limitType) {
      * {@inheritDoc}
      * @stable ICU 3.8
      */
+    @Override
     public String getType() {
         return "japanese";
```

Comment:
`CalType.JAPANESE.name()` ?

Reply:
Right. If I remember correctly, getType() was implemented before CalType enum.
I did not touch this method for this time - eclipse kindly :) supplied @Override annotation automatically when saving JapaneseCalendar.java changes.

Reply:
Other calendars have the same issue. I'd want to make consistent change with the new ticket  https://unicode-org.atlassian.net/projects/ICU/issues/ICU-20134